src/fides/api/util/errors.py

```diff
@@ -72,15 +72,40 @@ class ForbiddenError(HTTPException):
     To be raised when a user cannot modify an entity.
     """
 
-    def __init__(self, resource_type: str, fides_key: str) -> None:
+    def __init__(
+        self,
+        resource_type: str,
+        fides_key: str,
+        error_message: str = "user does not have permission to modify",
+    ) -> None:
         detail = {
-            "error": "user does not have permission to modify",
```

Comment:
This was the main issue - this error message couldn't be overridden with anything more relevant to more specific types of 403 errors.